config.yaml

```diff
@@ -2,19 +2,19 @@ options:
   calico-node-image:
     type: string
     # Please refer to layer-canal/versioning.md before changing the version below.
-    default: quay.io/calico/node:v2.6.12
+    default: docker.io/calico/node:v3.6.1
     description: |
       The image id to use for calico/node.
   calico-policy-image:
     type: string
-    default: quay.io/calico/kube-controllers:v1.0.5
+    default: docker.io/calico/kube-controllers:v3.6.1
     description: |
       The image id to use for calico/kube-controllers.
   ipip:
-    type: boolean
-    default: false
+    type: string
```

Comment:
What's going to happen here on upgrade from a boolean?

Reply:
If it's set to the default, it upgrades to the new default value just fine.

No idea what happens if you upgrade when the value is not the default. I'm guessing it shouldn't be catastrophic?

Reply:
Would that be an easy thing for you to test at this point?

Reply:
Gave it a quick test. It just reverts to the new default. That can go in a release note.